lib/membership_records.rb

```diff
@@ -0,0 +1,30 @@
+require 'open-uri'
+require 'open-uri/cached'
+require 'json'
+require_relative 'membership_record'
+
+OpenURI::Cache.cache_path = '.cache'
+
+class MembershipRecords < Scraped::JSON
+  field :members_of_the_lower_house do
+    lower_house_members.map do |mem|
+      fragment mem => MembershipRecord
+    end
+  end
+
+  field :next do
+    json[:next]
+  end
+
+  def json_for(url)
```

Comment:
What is this method? It seems quite out of place here, but I can't see where it's actually used.